admin/installer/_images.py

```diff
@@ -144,7 +143,7 @@ class PackerConfigure(PClass):
     :ivar template: The prototype configuration to use as a base. One of
         `docker` or `flocker`.
     :ivar distribution: The operating system distribution to install.
-        ubuntu-14.04 is the only one implemented so far.
+        ubuntu-16.04 is the only one implemented so far.
```

Comment:
This is really weird. It seems like we just change the hard coded default but keep it as a hard coded default. Can we get rid of this?